crates/re_types/definitions/rerun/blueprint/components/zoom_behavior.fbs

```diff
@@ -0,0 +1,29 @@
+include "arrow/attributes.fbs";
+include "python/attributes.fbs";
+include "rust/attributes.fbs";
+
+include "rerun/datatypes.fbs";
+include "rerun/attributes.fbs";
+
+namespace rerun.blueprint.components;
+
+// ---
+// TODO(#3384)
+/*
+enum ZoomBehavior: byte {
+    PreserveAspectRatio = 1,
+    LockToRange = 2,
+}
+*/
+
+/// How the axis behaves when the user zooms.
```

Comment:
related to the open question of behavior on single axis zoom, this should clearly indicate that it's about what happens to this axis when zoom is uniform.
open for debate how this should exactly relate